system/filesystem.py

```diff
@@ -41,6 +41,12 @@
     description:
     - If yes, allows to create new filesystem on devices that already has filesystem.
     required: false
+  resizefs:
+    choices: [ "yes", "no" ]
+    default: "no"
+    description:
+    - If yes, if the block device and filessytem size differ, grow the filesystem into the space. Note: XFS Will only grow if mounted.
```

Comment:
The build has failed because of this line. As you use ': ' symbol in description string, it becomes dict after YAML parsing in the process of generating documentation. You can check it in any yaml parses, for example [here](https://yaml-online-parser.appspot.com/). To fix this you can just put description line in quotes, or remove ': ' symbol.